README.md

```diff
@@ -231,6 +232,12 @@ other available gradle commands:
 
 - apply formatting: `./gradlew spotlessApply`
 
+## Join our community
+
+<a href="https://join.slack.com/t/composegears/shared_invite/zt-2noleve52-D~zrFPmC1cdhThsuQUW61A">
+    <img src="/assets/slack.png" alt="Slack" title="Slack" width="200">
+</a>
+
```

Comment:
```suggestion
```

Reply:
I believe we can remove this now, just use the badge instead.

Reply:
no, lets leave badge + section 🙂